app/controllers/bid_acceptances_controller.rb

```diff
@@ -1,9 +1,12 @@
 class BidAcceptancesController < ApplicationController
-	before_action :authenticate_broker_or_forward_freight!
+	before_action :authenticate_business_user!
 
   def create
     @bid = Bid.find(params[:bid_id])
     @bid.toggle_accept(@bid.accepted)
+    
+    @bid.accepted ? @bid.job.update(agent_id: @bid.bidder.id) : @bid.job.update(agent_id: nil)
```

Comment:
Ohh okay :slightly_smiling_face: and in the same code, since a bid table does not contain a bidder, would `bidder.id` work?